core/src/Dscp/Core/Foundation/Educator.hs

```diff
@@ -471,6 +472,14 @@ makeLenses ''PrivateTx
 makeLenses ''PrivateTxWitness
 makeLenses ''PrivateTxAux
 
+certGradeToPrivateTx :: Timestamp -> CertificateGrade -> PrivateTx
+certGradeToPrivateTx time CertificateGrade{..} =
+    PrivateTx
+    { _ptSignedSubmission = error "Mmm, what goes here???"
```

Comment:
As written in https://www.notion.so/serokell/Verifiable-PDF-certificates-6b49c78665134366934429be962f9880:
>As those certificates would not consist of actual submissions generated and sent to Educator by actual students, but rather of the submissions constructed by the certificate creator, all the submission signatures and student's address for such FairCV should be produced via dedicated secret key (e. g. consisting only from zeros/ones).

So, just hardcode some Student's private key and contents/assignments hashes for submissions. Another option is to use Educator's key to make signatures and call such grades "self-signed". However, this make it slightly harder for the users of certificate format (e.g. frontend) to distinguish between certificates produced with and without student's signatures.